nav2_collision_monitor/README.md

```diff
@@ -66,3 +67,9 @@ The following notes could be made:
 
  * Due to sheer speed, circle shapes are preferred for the approach behavior models if you can approximately model your robot as circular.
  * More points mean lower performance. Pointclouds could be culled or filtered before the Collision Monitor to improve performance.
+
+
+## Collision Detector
+
+Another node exists in the nav2_collision_monitor package called the Collision Detector. This node works similarly to the collision monitor
+except that it does not affect the robot's velocity. It will only inform that data from the configured sources has been detected within the configured polygons via message to the `/collision_detector_state` topic. 
```

Comment:
I think, this is also important to tell that unlike Collision Monitor that uses different behavior models, Collision Detector does not apply any of them. It detects the points inside given polygon instead, and reports back to user.